customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+    out.userButtonTitle = 'Zmień swoją nazwę użytkownika';
+    out.changeNamePrompt = 'Zmień swoją nazwę (Pozostaw puste, by być anonimowym): ';
+
+    out.renameButton = 'PRZEMIANUJ'; //Can be changed to "Zmień nazwę"
+    out.renameButtonTitle = 'Zmień nazwę pod którą plik wyświetli się na stronie głównej';
+    out.renamePrompt = 'Jak chciałbyś nazwać ten pad?';
+    out.renameConflict = 'Inny pad już ma taką nazwę';
+    out.clickToEdit = "Naciśnij by edytować";
+
+    out.forgetButton = 'ZAPOMNIJ';
+    out.forgetButtonTitle = 'Usuń ten dokument z listy wyświetlanej na stronie głównej';
+    out.forgetPrompt = 'Wciskając OK usuniesz ten URL z pamięci lokalnej, jesteś tego pewien?';
+
+    out.shareButton = 'Udostępnij';
+    out.shareButtonTitle = "Kopiuj URL do schowka";
+    out.shareSuccess = 'Pomyślnie skopiowano URL';
+    out.shareFailed = "Wystąpił błąd przy kopiowaniu";
+
+    out.presentButton = 'PREZENTUJ';
+    out.presentButtonTitle = "Otwórz tryb prezentacji";
+    out.presentSuccess = 'Naciśnij ESC aby wyjść z trybu prezentacji';
+    out.sourceButton = 'POKAŻ ŹRÓDŁO';
+    out.sourceButtonTitle = "Opuść tryb prezentacji";
+
+    out.backgroundButton = 'KOLOR TŁA';
+    out.backgroundButtonTitle = 'Zmień kolor tła dla tej prezentacji';
+    out.colorButton = 'KOLOR TEKSTU';
+    out.colorButtonTitle = 'Zmień kolor tekstu dla tej prezentacji';
+
+    out.commitButton = 'ZAPISZ'; //Can be discussed
+
+    out.getViewButton = 'URL tylko do odczytu';
+    out.getViewButtonTitle = 'Kopiuj URL dokumentu tylko do odczytu';
+    out.readonlyUrl = 'Dokument tylko do odczytu';
+    out.copyReadOnly = "Kopiuj URL do schowka";
+    out.openReadOnly = "Otwórz w nowej karcie";
+    out.editShare = "Udostępnij URL do edycji";
+    out.editShareTitle = "Zapisz URL do edycji w schowku";
+    out.viewShare = "Udostępnij URL tylko do odczytu";
+    out.viewShareTitle = "Zapisz URL tylko do odczytu w schowku";
+    out.viewOpen = "Otwórz podgląd w nowej karcie";
+    out.viewOpenTitle = "Otwórz ten dokument w nowej karcie, tylko do odczytu";
+
+    out.notifyJoined = "{0} dołączył do sesji współpracy";
+    out.notifyRenamed = "{0} jest teraz znany jako {1}";
+    out.notifyLeft = "{0} opuścił sesję współpracy";
+
+    out.disconnectAlert = 'Utracono połączenie sieciowe!';
+
+    out.tryIt = 'Wypróbuj!';
+    out.recentPads = 'Ostatnio otwarte dokumenty (przechowywane jedynie w twojej przeglądarce)';
+
+    out.okButton = 'OK (enter)';
+    out.cancelButton = 'Anuluj (esc)';
+
+    out.loginText = '<p>Twoja nazwa użytkownika i hasło są użyte do wygenerowania unikalnego klucza nie zapisanego na serwerze.</p>\n' +
+                    '<p>Uważaj by nie zapomnieć swoich danych, odzyskanie ich jest niemożliwe.</p>';
+
+    out.forget = "Zapomnij";
+
+    // Polls
+
+    out.poll_title = "Prosty koordynator planu dnia"; // Choice of "Koordynator" can be discussed
+    out.poll_subtitle = "Proste planowanie tygodnia, <em>w czasie rzeczywistym</em>";
```

Comment:
True, but this feature is used for voting for a possible meeting schedule (repeating), or a single meeting, and this shows who is able to attend at each possible time and date.
It might in that case be more appropriate to call this "koordynator spotkań" (meeting planner) and "Proste planowanie spotkań" (Simple meeting planning; this sounds better in Polish than English).